### PR TITLE
Move Extensions to alpha

### PR DESCRIPTION
**Pull request by kckst8**

### CODE REVIEW

**Comment by pmconne** [comment]
For those of us not in the loop: what is the motivation and what needs to happen before they can be promoted back to beta?

**Comment by kckst8** [comment]
> For those of us not in the loop: what is the motivation and what needs to happen before they can be promoted back to beta?

They aren't yet in a state where we are comfortable enough that there would not be breaking changes before the 3.1 release (which I'm told is expected early next week)

**Comment by calebmshafer** [comment]
> > For those of us not in the loop: what is the motivation and what needs to happen before they can be promoted back to beta?
> 
> They aren't yet in a state where we are comfortable enough that there would not be breaking changes before the 3.1 release (which I'm told is expected early next week)

@pmconne @kckst8 in addition we won't have any documentation or samples available initially. That along with the fact runtime Extensions haven't been implemented yet we felt we should move it down until those two things are covered.

File: common/changes/@itwin/core-frontend/extensions-alpha_2022-02-25-18-35.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/core-frontend",
+      "comment": "Move Extension APIs to alpha",
```
Comment (calebmshafer) [inline common/changes/@itwin/core-frontend/extensions-alpha_2022-02-25-18-35.json:5]: This shouldn't be needed as the Extension APIs were only just added in this release.

Comment (kckst8) [reply common/changes/@itwin/core-frontend/extensions-alpha_2022-02-25-18-35.json]: deleted

**Review by pmconne — APPROVED** [review]

File: common/changes/@itwin/core-frontend/extensions-alpha_2022-02-25-18-35.json (hunk 1 above)
Comment (aruniverse) [reply common/changes/@itwin/core-frontend/extensions-alpha_2022-02-25-18-35.json]: i think we still need the change log, just no comment

Comment (kckst8) [reply common/changes/@itwin/core-frontend/extensions-alpha_2022-02-25-18-35.json]: re-added

**Review by calebmshafer — APPROVED** [review]

**Pull request merged**